api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -122,6 +122,14 @@ message ExtAuthz {
   // :ref:`certificate<envoy_v3_api_field_service.auth.v3.AttributeContext.Peer.certificate>`.
   bool include_peer_certificate = 10;
 
+  // Specifies whether HTTP status codes for gRPC calls are passed intact from
+  // the called service back to the caller on gRPC DENIED.
+  //
+  // When this field is true, Envoy's HTTP response to gRPC calls which receive
+  // a DENIED response will retain the status code from the called service's
+  // HTTP headers. The gRPC headers are not affected.
```

Comment:
Which gRPC headers? What if the client call was gRPC? It's helpful to be very specific here.